noir-projects/noir-protocol-circuits/crates/types/src/meta/mod.nr

```diff
@@ -333,7 +333,12 @@ pub comptime fn generate_serialize_to_fields(
             let packed_struct_name = f"{name_at_one_token}_aux_var".quoted_contents();
 
             // We add the individual fields to the fields array
-            let packed_struct = quote { let $packed_struct_name = $name.pack() };
+            let pack_method = get_trait_impl_method(
+                typ,
+                quote { crate::traits::Packable<$maybe_packed_len_typ> },
```

Comment:
This doesn't fix the issue, but given that `packed_len` will have the solved number, we can use that.

```suggestion
                quote { crate::traits::Packable<$packed_len> },
```